src/pages/home/report/ReportActionItemSingle.js

```diff
@@ -67,7 +67,7 @@ const showUserDetails = (accountID) => {
 
 const ReportActionItemSingle = (props) => {
     const actorEmail = props.action.actorEmail.replace(CONST.REGEX.MERGED_ACCOUNT_PREFIX, '');
-    const {accountID, avatar, displayName, pendingFields} = props.personalDetails[actorEmail] || {};
+    const {accountID, avatar, displayName, pendingFields, login} = props.personalDetails[actorEmail] || {};
```

Comment:
What does `login` get used for?

Reply:
It is not, I missed this in the refactor. Reverting now